File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/3040.py
n = int(input())

for i in range(0, n):
    ns = input().split()
    a = int(ns[0])  
    b = int(ns[1])
    c = int(ns[2])
    if(a>=200 and a<=300 and b>=50 and c>=150):
        print("Sim")
    else:
        print("Nao")

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/3040.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')
    
n = parseInt(lines.shift())

for(i=0;i<n;i++){
    ns = lines.shift().split(' ')
    a = parseInt(ns[0])
    b = parseInt(ns[1])
    c = parseInt(ns[2])
    if(a>=200 && a<=300 && b>=50 && c>=150){
        console.log("Sim")
    } else{
        console.log("Nao")
    }
}
